shared/getDeviceOnDash/raspberrypi.md

```diff
@@ -1,10 +1,8 @@
-## Setting Up Your Device
-
-Insert the SD card into the {{ $device.name }} and, if necessary, connect the ethernet cable or the USB WiFi adapter. Now power up your Pi by inserting the micro USB cable.
+Insert the SD card into your {{ $device.name }} and, if necessary, connect the ethernet cable or the USB WiFi adapter. Now power up your Pi by inserting the micro USB cable.
 
 ![insert SD](/img/gifs/insert-sd.gif)
 
-It will take a minute or two for the device to appear on your [resin.io dashboard][resinDash]. While you wait the resinOS is expanding the partitions on your SD card to use all available space, installing a custom linux environment and establishing a secure connection with our servers.
+It will take a minute or two for the {{ $device.name }} to appear on your **resin.io** [dashboard][resinDash]. While you wait, the **resinOS** is expanding the partitions on your SD card to use all available space, installing a custom Linux environment, and establishing a secure connection with the **resin.io** servers.
```

Comment:
I think the "expanding the partitions on your SD card to use all available space" sounds better than the "partitioning your SD card" that most of the other guides use :-)